src/migrations/001_initial.ts

```diff
@@ -0,0 +1,60 @@
+import { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+
+  if (await knex.schema.hasTable('changelog')) {
+    // This migration has been applied
+    return;
+  }
+
+  await knex.schema.createTable('changelog', table => {
+
+    table.integer('id').unsigned().primary();
+    table.integer('timestamp').unsigned();
+
+  });
+
+  await knex('changelog').insert({
+    id: 1,
+    timestamp: Math.floor(Date.now()/1000)
+  });
+
+  await knex.schema.createTable('users', table => {
```

Comment:
@mihok This is an example of a migration that I edited to work with postgres. I only did path 001 to 004 so far.